### DISCUSSION TITLE
"index"/"worktree" for "staged"/"unstaged" when stashing is confusing

### DISCUSSION THREAD

**Discussion by verhovsky** [magit/magit#4647] [Ideas]
I have been using magit for a couple years and git for longer than that and when stashing changes, I still cannot remember whether "worktree" are staged files or unstaged files (is it as in files I'm "working on" whereas as opposed to staged files are files I'm done working on and ready to commit, or is "index" a better term for that?). I have to think about it every time and frankly I still feel unsure. I don't think there's a good reason to have a different pair of words for this same concept. Is this some internal git term? I googled it just now and there's something called `git-worktree` and the first search result is someone asking "what do I use this for", i.e. "I don't use this" and neither do I.

Could I get a setting that will just rename them in the UI?

**Comment by tarsius** [comment]
Magit inherits these terms from Git, yes. I am going to stick to them and won't add an option to use other terms. Sorry.
